src/web/mjs/connectors/AssortedScans.mjs

```diff
@@ -0,0 +1,65 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs'
+
+export default class AssortedScans extends Connector {
+    constructor() {
+        super();
+        super.id = 'assortedscans';
+        super.label = 'assortedscans';
+        this.tags = ['manga', 'english'];
+        this.url = 'https://assortedscans.com/';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '#series-title');
+        let title = data.text.trim();
```

Comment:
Was this tested? I think data is an array and therefore this would throw an exception...